### ISSUE TITLE
MouseKeys: Horizontal scroll doesn't work under Linux

### ISSUE THREAD

**Issue by algernon** [keyboardio/Kaleidoscope#373]
Originally by @algernon, as keyboardio/Kaleidoscope-MouseKeys#24.

**Comment by algernon** [comment]
@jamadagni I can't easily test on Bionic, but I'll try in a VM just in case. Meanwhile, could you run `evemu-record`, and see what it says when you press the horizontal scroll keys?

Mine says:

```
E: 123.589492 0002 0006 -001    # EV_REL / REL_HWHEEL           -1
E: 123.589492 0000 0000 0000    # ------------ SYN_REPORT (0) ---------- +116658ms
E: 123.641523 0002 0006 -001    # EV_REL / REL_HWHEEL           -1
E: 123.641523 0000 0000 0000    # ------------ SYN_REPORT (0) ---------- +52ms
E: 124.369469 0002 0006 0001    # EV_REL / REL_HWHEEL           1
E: 124.369469 0000 0000 0000    # ------------ SYN_REPORT (0) ---------- +728ms
E: 124.420479 0002 0006 0001    # EV_REL / REL_HWHEEL           1
E: 124.420479 0000 0000 0000    # ------------ SYN_REPORT (0) ---------- +51ms
```

**Comment by algernon** [comment]
Ok. I spun up a Kubuntu Bionic VM, and indeed, horizontal scroll doesn't work (I made sure that the devices I test with are on USB passthrough). It reports as `REL_DIAL`, like in the original report.

I tried another device, which has a built-in horizontal wheel (but sadly no open source firmware), and that doesn't work in Bionic either.

I assume that there's something in the kernel, libinput, or Xorg that changed since Bionic, which made horizontal wheel work, but I do not know which component. In any case, it looks like there's nothing we can do on the firmware side to fix hWheel, as our side appears correct.

I'm afraid I'll have to close this issue, as the solution is not on the firmware side.
